extract/extract/extractors/zalando_de.py

```diff
@@ -135,32 +138,47 @@ def extract_zalando(
         return None
 
 
-def get_sustainability_strings(beautiful_soup: BeautifulSoup) -> Iterator[str]:
+def get_json_data(json_file: str) -> Any:
+    """
+    Helper function to parse mangled json files.
+
+    Args:
+        json_file (str): Either a plain json file or a json file encoded as an xml CDATA string
+
+    Returns:
+        Iterator[str]: found sustainability strings
+    """
+
+    try:
+        return json.loads(json_file)
+    except json.decoder.JSONDecodeError:
+        json_file = ElementTree.fromstring(f"<root>{json_file}</root>").text
+        return json.loads(json_file)
+
+
+def get_sustainability_strings(parsed_page: ParsedPage) -> Iterator[str]:
```

Comment:
Since the current code does not need `parsed_page` revert this please to use the original `BeautifulSoup` object.